src/components/TransactionList/TransactionRow/TransactionRowTs.ts

```diff
@@ -124,7 +124,7 @@ export class TransactionRowTs extends Vue {
      * Current transaction Details
      * @type {AggregateTransaction}
      */
-    private aggregateTransactionDetails: AggregateTransaction;
+    private aggregateTransactionDetails: AggregateTransaction = null;
```

Comment:
Can you elaborate on this change?

Reply:
I needed to reference that in `TransactionRow.vue` file so It requires to have a value declared.